tests/Backend/Snowflake/CloneIntoWorkspaceTest.php

```diff
@@ -112,6 +112,19 @@ public function testClone($aliasNestingLevel): void
         $workspaceTableData = $backend->fetchAll('languagesDetails');
         $this->assertCount(5, $workspaceTableData);
 
+        // try to import table from workspace
+        // table languagesDetails has _timestamp column
+        // but import still works
+        $this->_client->writeTableAsyncDirect(
```

Comment:
Keboola\StorageApi\ClientException : The table languages-0 is an alias table. Import is disabled. Please import data into the source table languagesDetails.
 /code/src/Keboola/StorageApi/Client.php:2777
 /code/src/Keboola/StorageApi/Client.php:2671
 /code/src/Keboola/StorageApi/Client.php:1155
 /code/tests/Backend/Snowflake/CloneIntoWorkspaceTest.php:121

Reply:
fixed